modules/puzzle/src/main/Daily.scala

```diff
@@ -51,8 +51,8 @@ private[puzzle] final class Daily(
   ).uno[Puzzle]
 
   private def findNew = coll.find(
-    $doc(F.day $exists false)
-  ).sort($doc(F.voteSum -> -1)).uno[Puzzle] flatMap {
+    $doc(F.day $exists false, F.voteEnabled -> true)
```

Comment:
so... we're just picking *any* puzzle that's not disabled. Instead of the most upvoted. It compiles, but is hardly a good change.